src/js/lib/learning_resources.js

```diff
@@ -0,0 +1,52 @@
+import moment from "moment"
+
+import {
+  COURSE_ARCHIVED,
+  COURSE_AVAILABLE_NOW,
+  COURSE_CURRENT,
+  COURSE_PRIOR,
+  DATE_FORMAT,
+  LR_TYPE_USERLIST,
+  LR_TYPE_LEARNINGPATH,
+  LR_TYPE_PROGRAM,
+  LR_TYPE_COURSE,
+  platforms
```

Comment:
Many of these don't seem to be used here

Reply:
you're right - the `no-unused-vars` lint rule was turned off for some reason